src/Administration/Resources/app/administration/src/module/sw-product/component/sw-product-properties/sw-product-properties.html.twig

```diff
@@ -131,7 +131,9 @@
                                     :dismissable="acl.can('product.deleter')"
                                     @dismiss="onDeletePropertyValue(option)"
                                 >
-                                    {{ option.translated.name }}
+                                    <template
+                                        v-html="option.translated.name"
```

Comment:
v-html does not execute JS code right? also all `on-` listeners are not working?

Reply:
Good point. Vue is as safe as the Browser as they do not ship any sanitizer. But you do. So I used it instead